app/helpers/change_request_helper.rb

```diff
@@ -8,9 +8,9 @@ def request_state(change_request)
   end
 
   def applicant_response(change_request)
-    if change_request.class.name == "DescriptionChangeRequest"
+    if change_request.class.name == "DescriptionChangeRequest" || change_request.class.name == "RedLineBoundaryChangeRequest"
       if change_request.approved?
-        "Description change has been approved by the applicant"
+        "Change has been approved by the applicant"
```

Comment:
just double checking we run this past Ali/Tom? (I can\t

ALI blocked a ticket from from being approved before due to a tiny text change so just wanted to make sure she doesn't want different text being displayed depending on whether it is a red line boundary or a description change that has been approved - I don't see a design for the index text showing in the ticket so just best to double check